cypress/e2e/admin/administrateMigration.feature

```diff
@@ -0,0 +1,26 @@
+@release
+Feature: Admin Users - To administrate the Migration process.
+
+  As an Admin I want to start, finish and make the migration mandatory.
+
+  Background:
+    Given I am logged in as a 'admin' at 'nbc'
+    When I go to administration page
+    When I go to school administration
+
+  Scenario: Admin starts the migration and the School Number is added
+    When I can see the school number
+    Then I can click on the start migration button
```

Comment:
"Then I see button Start migration is enabled" can be better understood if someone is reading the feature file without checking the methods.

Can + activity (click) doesn't make clear what will be done in this step.

Reply:
Thank you for the detailed comments, it's corrected.